packages/core/utils/package.json

```diff
@@ -46,7 +46,7 @@
     "prepublishOnly": "cross-env NODE_ENV=production tsc --build",
     "build": "rimraf dist && tsc --build",
     "watch": "tsc --build --watch",
-    "test": "jest --silent --bail --maxWorkers=50% --forceExit --testPathIgnorePatterns='/integration-tests/' -- src/**/__tests__/**/*.ts",
+    "test": "jest --silent=false --bail --maxWorkers=50% --forceExit --testPathIgnorePatterns='/integration-tests/' -- src/**/__tests__/**/*.ts",
```

Comment:
suggestion: can you revert this please 😄

Reply:
haha. Just curious, why do we silent logs by default?

Reply:
because it was creating a lot of noise in the pipeline and becoming infinite scroll, sometimes bugging the view 😂

Reply:
it also slowdown the pipeline